pkg/api/api.go

```diff
@@ -436,5 +436,8 @@ func (hs *HTTPServer) registerRoutes() {
 	r.Get("/api/snapshots-delete/:deleteKey", reqSnapshotPublicModeOrSignedIn, Wrap(DeleteDashboardSnapshotByDeleteKey))
 	r.Delete("/api/snapshots/:key", reqEditorRole, Wrap(DeleteDashboardSnapshot))
 
+	// Frontend logs
+	r.Post("/log_frontend", bind(FrontendSentryEvent{}), Wrap(hs.LogFrontendMessage))
```

Comment:
I'm not wild about this endpoint... the only `_` in any URL... maybe just "/log"?  or "/log/frontend"?

Reply:
`/log` sounds good, don't think there's any scenario to log non frontend stuff
- [ ] rename endpoint